sdks/python/setup.py

```diff
@@ -147,7 +147,7 @@ def get_version():
     # server, therefore list of allowed versions is very narrow.
     # See: https://github.com/uqfoundation/dill/issues/341.
     'dill>=0.3.1.1,<0.3.2',
-    'fastavro>=0.21.4,<0.22',
+    'fastavro>=0.21.4,<0.23',
```

Comment:
Should we up this to <0.24 ?

Reply:
Feel free to take over the MR or provide an alternative. I haven't got enough knowledge of beam nor fastavro to fix the test that failed.

Reply:
+1, let's use `<0.24`